core/bourbon/settings/_settings.scss

```diff
@@ -34,10 +35,31 @@ $_bourbon-defaults: (
   "rails-asset-pipeline": false,
 );
 
-/// User overrides of Bourbon configuration settings.
+/// Global Bourbon settings.
+///
+/// @name Settings
 ///
 /// @type map
 ///
+/// @property {color} contrast-switch-dark-color [#000]
+///   Global dark color for the `contrast-switch` function.
+///
+/// @property {color} contrast-switch-light-color [#fff]
+///   Global light color for the `contrast-switch` function.
+///
+/// @property {list} global-font-file-formats [("ttf", "woff2", "woff")]
+///   Global font file formats for the `font-face` mixin.
+///
+/// @property {number (with unit)} modular-scale-base [1em]
+///   Global base value for the `modular-scale` function.
```

Comment:
Units in Sass/CSS are things like `em`, `rem`, etc. So I think value makes sense here, because we are setting `1em`, not just `em`.